pkg/controller/nodelifecycle/scheduler/taint_manager.go

```diff
@@ -156,15 +156,16 @@ func getMinTolerationTime(tolerations []v1.Toleration) time.Duration {
 // NewNoExecuteTaintManager creates a new NoExecuteTaintManager that will use passed clientset to
 // communicate with the API server.
 func NewNoExecuteTaintManager(ctx context.Context, c clientset.Interface, getPod GetPodFunc, getNode GetNodeFunc, getPodsAssignedToNode GetPodsByNodeNameFunc) *NoExecuteTaintManager {
+	if c == nil {
+		klog.Fatalf("kubeClient is nil when starting NodeController")
+	}
+
 	eventBroadcaster := record.NewBroadcaster()
 	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "taint-controller"})
 	eventBroadcaster.StartStructuredLogging(0)
-	if c != nil {
-		klog.V(0).InfoS("Sending events to api server")
-		eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: c.CoreV1().Events("")})
-	} else {
-		klog.Fatalf("kubeClient is nil when starting NodeController")
-	}
+
+	klog.V(0).InfoS("Sending events to api server")
```

Comment:
I know that this is not related to your MR but I would suggest changing this to:
```suggestion
	klog.Info("Sending events to api server")
```

Reply:
It doesn't make a lot of sense.

Reply:
Why?

Reply:
I think they behave the same, no performance improvement or better expression and it's better to keep the consistent style(we use `klog.V(x)` for the whole package). WDYT?

Reply:
Ah, you are right. It seems like the `klog.V(x)` is used across this file (probably package as well, as you mentioned). Missed that.

Reply:
So let's better leave it as is.